src/node/rpc/memberfrontend.h

```diff
@@ -361,6 +361,13 @@ namespace ccf
       return false;
     }
 
+    void record_voting_history(
```

Comment:
I was originally thinking that this could be done generically in `frontend.h` if the handler was registered as "record voting history" but I don't think it is something we want to expose to app developers.

Reply:
We could have a variant for the user fronted, because although it goes to a different table, we also support signing and storing requests for audit.

Reply:
True. Is this something that we want to implement now or delay until users have asked for it (if they haven't already!)?

Reply:
Ticket and delay I think.